workflow/applications/applications.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, conf: Configuration) -> None:
         self.do_genesis = _base.get('DO_GENESIS', True)
         self.do_genesis_fsu = _base.get('DO_GENESIS_FSU', False)
         self.do_metp = _base.get('DO_METP', False)
+        self.do_npoess = _base.get('DO_NPOESS', False)
```

Comment:
Where is `DO_NPOESS` in `config.base`?

Reply:
Thank you. I forgot to push this update following testing.